### ISSUE TITLE
OCLI - ocli --help is not displaying any help about subcommands supported

### ISSUE THREAD

**Issue by piniguez** [open-mpi/orcm#150]
when <pre>ocli --help</pre> is executed it displays no information about sub-commands supported:
  - resource
  - queue
  - session...

The following is the current output of ocli:
<pre>
[pedro@karate bin]$ ./ocli --help
Usage: ocli [OPTIONS]
  Open Resilient Cluster Manager "ocli" Tool

   -am <arg0>            Aggregate MCA parameter set file list
   -gomca|--gomca <arg0> <arg1>
                         Pass global MCA parameters that are applicable to
                         all contexts (arg0 is the parameter name; arg1 is
                         the parameter value)
-h|--help                This help message
   -omca|--omca <arg0> <arg1>
                         Pass context-specific MCA parameters; they are
                         considered global if --gomca is not used and only
                         one context is specified (arg0 is the parameter
                         name; arg1 is the parameter value)
-v|--verbose             Be Verbose
-V|--version             Show version information
</pre>

**Comment by benmcclelland** [comment]
I don't think we want to clutter up the help output with all the subcommands.  They are documented in the man pages and wiki, so maybe I can add a link to those in the help output.  This help was intended to document the options for the octl and ocli commands themselves (ie: ability to take --omca paramters).

**Comment by piniguez** [comment]
ok, so do you want me to close this issue?

I opened it because I tried to use the help from command line to understand how to use ocli (which is something I do for many other commands like grep, vi,..) but I found nothing helpful. After checking the wiki I finally understood how to use it.
